OxfordBodleian/Juel-Jensen/BDLaethf30.xml

```diff
@@ -149,10 +136,535 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                     elements of punctuation marks.
+                                 </seg>
+                                 <list type="abbreviations">
+                                     <item>single letters for the entire refrain in <ref target="#p1_i1.4"/>;</item>
+                                     <item>
+                                         <abbr>ሰ፡</abbr>, <abbr>ሰአ፡</abbr> or <abbr>ሰአሊ፡</abbr> for <expan>ሰአሊ፡ ለነ፡ ቅድስት፡</expan> (e.g. <locus target="#140vb #142rb #144vb"/>)
+                                     </item>
+                                 </list>
+                             </handNote>
+                         </handDesc>
+                     
+                        
+                        <additions>
+                            <list>
+                                <item xml:id="e1">
+                                    <locus target="#ir"/>
+                                    <desc>A white sticker is glued to <locus target="#ir"/>. The name of <persName ref="PRS5782JuelJen"/> is printed
+                                        on it
+                                        (<foreign xml:lang="gez">ቤንት፡ ዩውል፡ የንሰን።</foreign>) and the signatures "MS. Aeth. f. 30" and 
+                                        "MS Et 5" are written in pencil.
+                                        An additional sticker, with the signature "Ms 5" on it, is glued below it.</desc>
+                                </item>
+                                
+                                <item xml:id="e2">
+                                    <locus target="#ir"/>
+                                    <desc>Recent note written in pencil and barely legible, containing minimal descriptive information on the manuscript.</desc>
+                                    <q xml:lang="en">Ethiopic <gap reason="illegible"/> Psalter followed by usual odes 
+                                        <gap reason="illegible"/>
+                                    late C 19 early C 20</q>
+                                </item>
+                            </list>
+                        </additions>
+                        
+                     <!--   <bindingDesc>
+                            <binding xml:id="binding">
+                                <decoNote xml:id="b1"></decoNote>
+                                <decoNote xml:id="b2" type="bindingMaterial">
+                           <material key="wood"/>
+                        </decoNote>
+                        leaf string marker f.  39, 101, 104
+                            </binding>
+                        </bindingDesc>-->
+                        
+                    </physDesc>
+                    
+                    <history>
+                        <p>Both units seem to be written by the same hand. However, <ref target="#p2"/> features different rubrication patterns
```

Comment:
I can't say for sure whether the manuscript has one or two production units. The hand looks the same to me as well, as does the nuance of the red ink, although the size of the characters is quite different between f. 147v and 148r. It is true that the rubrication pattern also is a bit different in the Anqāṣa bǝrhān, but this is a separate text, so this could be an intentional choice. The number of lines (22 at the end of the first unit, 21 in the second unit) is not that different and might depend on fluctuations between different quires. In general, the size of the leaves looks the same to me between the two proposed units. What do you think @ElianaDalSasso ?

Reply:
I can't say either, but I too think that the shade of red ink is the same. The leaves are terribly damaged but could be slightly smaller. I notice the same differences in the in the size of the characters (do they start on the last line of f. 147v already?). The sewing is damaged and repaired so it is difficult to judge if the leaves have been added at a later stage...now they seems to have been coarsely sewn together and added to the bookblock...but it is a common situation of the last leaves of a ms.
I've just noticed that the last leaves seems to be single leaves (see inner margin of f. 148 photo 149). Does this influence the decision? Maybe not.

Reply:
Good, then we can stick to the current structure of the MS description and postpone further discussion once the manuscript is consulted again.

Reply:
I think that checking again the manuscript would be the best solution!